ledger/participant-integration-api/src/test/lib/scala/platform/store/dao/JdbcLedgerDaoBackend.scala

```diff
@@ -55,6 +54,7 @@ private[dao] trait JdbcLedgerDaoBackend extends AkkaBeforeAndAfterAll { this: Su
   override protected def beforeAll(): Unit = {
     super.beforeAll()
     implicit val executionContext: ExecutionContext = system.dispatcher
+    implicit val resourceContext: ResourceContext = ResourceContext(executionContext)
```

Comment:
The double implicit execution context here seems like asking for trouble. Since this PR is already giant it would probably make sense to use the `ResourceContext` in `AkkaBeforeAndAfterAll`. You can tackle it in a second PR if you think it's a better idea, but it would probably make sense to make the change ASAP.

Reply:
That's a good idea. I'll make that change.

Reply:
Decided against this and went with the helper trait for these too, because otherwise `AkkaBeforeAndAfterAll` starts to depend on stuff in `//ledger`.